frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import _ from "underscore";
+import { t } from "ttag";
+
+import PinnedItemCard from "metabase/collections/components/PinnedItemCard/PinnedItemCard";
+import { Item, Collection } from "metabase/collections/utils";
+
+import { Container, Grid, SectionHeader } from "./PinnedItemOverview.styled";
+
+type Props = {
+  items: Item[];
+  collection: Collection;
+  onCopy: (items: Item[]) => void;
+  onMove: (items: Item[]) => void;
+};
+
+function PinnedItemOverview({ items, collection, onCopy, onMove }: Props) {
+  const sortedItems = _.sortBy(items, item => item.name);
+  const {
+    card: cardItems = [],
+    dashboard: dashboardItems = [],
+    dataset: datasetItems = [],
+  } = _.groupBy(sortedItems, "model");
+
+  return items.length ? (
+    <Container data-testid="pinned-items">
+      {cardItems.length > 0 && (
+        <Grid>
+          {cardItems.map(item => (
+            <PinnedItemCard
+              key={item.id}
+              item={item}
+              collection={collection}
+              onCopy={onCopy}
+              onMove={onMove}
+            />
+          ))}
+        </Grid>
+      )}
+      {dashboardItems.length > 0 && (
+        <Grid>
+          {dashboardItems.map(item => (
+            <PinnedItemCard
+              key={item.id}
+              item={item}
+              collection={collection}
+              onCopy={onCopy}
+              onMove={onMove}
+            />
+          ))}
+        </Grid>
+      )}
+      {datasetItems.length > 0 && (
+        <div>
+          <SectionHeader>
+            <h4>{t`Datasets`}</h4>
+            <div>{t`Start new explorations about {${collection.name}} here`}</div>
```

Comment:
Looks like the first and last brackets are redundant here: `{${collection.name}}`

![CleanShot 2022-01-20 at 13 37 04@2x](https://user-images.githubusercontent.com/17258145/150333221-7cbad089-0423-412f-8b45-4d40fbd0e068.png)

Reply:
this is, interestingly, something from the mocks. @mazameli do you want the brackets?

<img width="254" alt="Screen Shot 2022-01-20 at 9 39 34 AM" src="https://user-images.githubusercontent.com/13057258/150392590-ab164aa3-272f-4023-b47a-d187a217a18b.png">

Reply:
Sorry, that was meant to be a variable for `{collection_name}` so it should resolve to e.g. `Start new explorations about Revenue here` or `Start new explorations about Growth here`. I think we should special case the root/Our Analytics collection though: `Start new explorations here`, otherwise it'll read oddly.